telco-hub/configuration/reference-crs/required/gitops/ztp-installation/argocd-openshift-gitops-patch.json

```diff
@@ -0,0 +1,92 @@
+{
+  "spec": {
+    "controller": {
+      "resources": {
+        "limits": {
+          "cpu": "16",
+          "memory": "32Gi"
+        },
+        "requests": {
+          "cpu": "1",
+          "memory": "2Gi"
+        }
+      }
+    },
+    "kustomizeBuildOptions": "--enable-alpha-plugins",
+    "repo": {
+      "volumes": [
+        {
+          "name": "kustomize",
+          "emptyDir": {}
+        }
+      ],
+      "initContainers": [
+        {
+          "resources": {
+          },
+          "terminationMessagePath": "/dev/termination-log",
+          "name": "kustomize-plugin",
+          "command": [
+            "/exportkustomize.sh"
+          ],
+          "args": [
+            "/.config"
+          ],
+          "imagePullPolicy": "Always",
+          "volumeMounts": [
+            {
+              "name": "kustomize",
+              "mountPath": "/.config"
+            }
+          ],
+          "terminationMessagePolicy": "File",
+          "image": "registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.16.2"
+        },
+        {
+          "args": [
+            "-c",
+            "mkdir -p /.config/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator && cp /policy-generator/PolicyGenerator-not-fips-compliant /.config/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator"
+          ],
+          "command": [
+            "/bin/bash"
+          ],
+          "image": "registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel9:v2.10",
```

Comment:
Should be v2.11 until 2.12 comes out.

Reply:
right, I re-run the script to generate the manifests for 4.17. As today, ZTP 4.17 is going with 2.11. 
I suggest keep it as it is generated by ztp-site-generate today. So, we only run the script and we dont have think on versions. ztp-site-generate will know.
Actually, the doc of ztp 4.17, today, still points to ACM2.11
Anyway, I have updated all the manifests from 4.16 to 4.17.

Reply:
I saw that too. I've updated the docs to include ACM 2.12. We need to update the CRs too.

Reply:
do you want me to manually update it here? meanwhile is fixed on the source...